src/api.js

```diff
@@ -1,9 +1,11 @@
-// import axios from "axios";
+import axios from "axios";
 
```

Comment:
take a look into creating a baseURL for axios it could save you time as you make more api requests.